doc/sphinx/mgmt_preview_quickstart.rst

```diff
@@ -0,0 +1,152 @@
+Quickstart Tutorial - Resource Management (Preview Libraries)
+===============================================================
+
+We are excited to announce that a new set of management libraries are now in Public Preview.
+Those packages share a number of new features such as Azure Identity support,
+HTTP pipeline, error-handling.,etc, and they also follow the new Azure SDK guidelines which
+create easy-to-use APIs that are idiomatic, compatible, and dependable.
+
+You can find the details of those new libraries `here <https://azure.github.io/azure-sdk/releases/latest/#python>`__
+
+In this basic quickstart guide, we will walk you through how to
+authenticate to Azure using the preview libraries and start interacting with
+Azure resources. There are several possible approaches to
+authentication. This document illustrates the most common scenario
+
+Prerequisites
+-------------
+
+| You will need the following values to authenticate to Azure
+ 
+- **Subscription ID**
+- **Client ID** 
+- **Client Secret** 
+- **Tenant ID** 
+
```

Comment:
I was also thinking that we could add alternative of using Azure CLI as well. Maybe simpler for some users. Just "az login" and the same examples should work considering we use default credentials?